jaeger-operator.version

```diff
@@ -0,0 +1,3 @@
+# this is the latest released version of the jaeger-operator
```

Comment:
Shouldn't this file include a statement like "# DO NOT EDIT - this file is automatically updated by the release.sh"